splash/engines/chromium/webpage.py

```diff
@@ -4,6 +4,8 @@
 
 
 class ChromiumWebPage(QWebEnginePage):
+    http2_enabled = True
```

Comment:
I’m not sure if this is needed. The Webkit counterpart has a few default values specified like this.

Reply:
I think it is not needed here. In webkit counterpart it is used in these places:

* https://github.com/scrapinghub/splash/blob/86b0f934f17d4118ba4fd7c359e6d322dfaf7f74/splash/engines/webkit/browser_tab.py#L216
* https://github.com/scrapinghub/splash/blob/86b0f934f17d4118ba4fd7c359e6d322dfaf7f74/splash/network_manager.py#L328